framework/db/QueryBuilder.php

```diff
@@ -802,6 +802,34 @@ public function dropCommentFromTable($table)
     }
 
     /**
+     * Creates an SQL View.
+     *
+     * @param string $viewName the name of the view to be created.
+     * @param string|Query $subquery the select statement which defines the view.
+     * This can be either a string or a [[Query]] object.
+     * @return string the `CREATE VIEW` SQL statement.
+     */
+    public function createView($viewName, $subquery)
+    {
+        if ($subquery instanceof Query) {
+            list($subquery, $params) = $this->build($subquery);
+            // TODO how to deal with params here?
```

Comment:
I created a class which invokes `Query::createCommand()`.

https://github.com/tecnocen-com/yii2-migrate/blob/master/src/CreateViewMigration.php#L22

It works perfectly so far.